vendor/bat-native-ledger/src/bat/ledger/internal/gemini/gemini.cc

```diff
@@ -174,7 +156,7 @@ void Gemini::StartTransferFeeTimer(const std::string& fee_id,
                                    const int attempts) {
   DCHECK(!fee_id.empty());
 
-  base::TimeDelta delay = util::GetRandomizedDelay(base::Seconds(45));
+  base::TimeDelta delay = base::Minutes(5);
```

Comment:
why remove randomization here?

Reply:
That's going away for all wallet providers, as there's no privacy benefit of the random delay here (confirmed by @evq). Related GitHub issue is https://github.com/brave/brave-browser/issues/25089.